### DISCUSSION TITLE
Grid Template Rows Seems to not work

### DISCUSSION THREAD

**Discussion by tchiadeu** [tailwindlabs/tailwindcss#13697] [Help]
Hello,

I'm sorry if I don't post this at the good place.
I'm not really blocked but maybe this case is a bug :

I want to use the class `grid-rows-8`
According to the doc : "By default, Tailwind includes grid-template-row utilities for creating basic grids with up to 12 equal width rows."
But for me, if I try to have more than 6 rows, it doesn't work.
I have to add a new `gridTemplateRows` if my `tailwind.config.js` which have this option : `'8': 'repeat(8, minmax(0, 1fr))'`

Maybe I missed something somewhere explaining why this isn't working by default for me.

Thanks for the answer.

**Comment by wongjn** [comment]
Consider checking you are running Tailwind version `v3.4.0` or higher. `grid-rows-*` classes beyond 6 were added in this version, as per #12180.

Otherwise, please consider providing a [Tailwind Play](https://play.tailwindcss.com) or a project that reproduces the unexpected behavior.
